config/overlays/kubeflow/namespace-patch.yaml

```diff
@@ -0,0 +1,8 @@
+# This keeps the kubeflow namespace labels how they were initially.
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: kubeflow
+    istio-injection: enabled
```

Comment:
quick question, will the `istio-injection: enabled` apply to single user kubeflow when it runs on Kubeflow namespace? Or kubeflow/manifests has another overlay to overwrite this for single user setup?

Reply:
The intent behind this patch is to return to what the  `kubeflow` namespace is deployed as [here](https://github.com/kubeflow/manifests/blob/master/common/namespaces/base/namespaces.yaml). However, I just looked more into kustomize, and apparently it's possible to just remove the namespace resource from the generated YAML with a `$patch: delete`. This  way the kubeflow namespace won't be touched and is assumed to already exist.

Reply:
Yeah that namespace.yaml in the manifests is only used in the multi-user mode.